photon-client/src/components/dashboard/tabs/TargetsTab.vue

```diff
@@ -2,8 +2,38 @@
 import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
 import { PipelineType } from "@/types/PipelineTypes";
 import { useStateStore } from "@/stores/StateStore";
+import Vue, { ref, watch } from "vue";
+import type { Transform3d } from "@/types/PhotonTrackingTypes";
 
-const currentPipelineSettings = useCameraSettingsStore().currentPipelineSettings;
+let oldResults: {targets: Transform3d[]} = Vue.observable({ targets: [] });
```

Comment:
Don't think this needs to be like this, I'm pretty sure you can just do `const targets = []`